packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/add-connector/add-connector.html

```diff
@@ -0,0 +1,91 @@
+<oui-back-button
+    data-previous-page="{{:: 'pci_databases_connectors_add_back' | translate }}"
+    data-on-click="$ctrl.goBack()"
+></oui-back-button>
+
+<!-- error messages -->
+<cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+
+<div class="d-flex justify-content-between align-items-center mb-4">
+    <div>
+        <h2 ng-bind="$ctrl.availableConnector.name" class="mb-0"></h2>
+        <span class="oui-paragraph">
+            Version: {{$ctrl.availableConnector.version}} | Author:
+            {{$ctrl.availableConnector.author}} |
+            <a
+                ng-href="{{$ctrl.availableConnector.documentationUrl}}"
+                target="_blank"
+                rel="noopener"
+                >docs</a
+            >
+        </span>
+    </div>
+    <oui-button variant="primary" size="l" ng-click="$ctrl.addConnector()">
+        <span data-translate="pci_databases_connectors_add_button"></span>
+    </oui-button>
+</div>
+
+<oui-tabs>
+    <oui-tabs-item
+        ng-repeat="group in $ctrl.availableConnector.configuration.getGroups()"
+        heading="{{group.name}}"
+        subheading="{{group.required ? 'required' : ''}}"
+        variant="promotion"
+    >
+        <div class="row">
+            <div class="col-md-6">
+                <div
+                    ng-if="group.name === 'Transforms'"
```

Comment:
You have defined a enum in constants file for all values of group.name. Can you use it here plz

Reply:
It is done on the edit PR, which will be merged into this one: https://github.com/ovh/manager/pull/6579/commits/debbe74e2325235ffa6c315392f168d90f7d7ca7